tests/io/v3/vm/test_agent_groups.py

```diff
@@ -0,0 +1,186 @@
+'''
+test plugins
+'''
+import re
+
+import pytest
+import responses
+
+BASE_URL = 'https://cloud.tenable.com/api/v3/agent-groups'
+
+
+@responses.activate
+def test_add_agent_with_single_agent_id(api):
+    '''Test case for add_agent method with sigle agent id'''
+    group_id: int = 470009
+    agent_id: int = 5
```

Comment:
type should be UUID everywhere for group_id and agent_id

Reply:
Done.